source/predicates/number.ts

```diff
@@ -82,6 +82,27 @@ export class NumberPredicate extends Predicate<number> {
 		});
 	}
 
+	/**
+	 * Test if a number is an element of the provided list.
+	 *
+	 * @param list List of possible values.
+	 */
+	oneOf(list: number[]) {
+		return this.addValidator({
+			message: (value, label) => {
+				let printedList = JSON.stringify(list);
+
+				if (list.length > 10) {
+					const overflow = list.length - 10;
+					printedList = JSON.stringify(list.slice(0, 10)).replace(/]$/, `,…+${overflow} more]`);
+				}
+
+				return `Expected ${label} to be one of \`${printedList}\`, got ${value}`;
```

Comment:
In the `string` predicate, the value in `got ${value}` is wrapped in backtics, since this was not done in other validators in the `number` predicate, I removed them here.

Reply:
I think we should align this with the other predicates, but do it in a separate PR.